pkg/controller/perconaservermongodb/smart.go

```diff
@@ -58,12 +59,12 @@ func (r *ReconcilePerconaServerMongoDB) smartUpdate(cr *api.PerconaServerMongoDB
 		return nil
 	}
 
-	ok, err := r.isBackupRunning(cr)
+	ok, err := backup.HasActiveJobs(r.client, cr, backup.Job{})
```

Comment:
can we please don't completely remove the current `isBackupRunning` but in addition to `isRestoreRunning` and to `isBackupRunning` also run `HasActiveJobs` check?

Reply:
added it back in dbb7b7b3177ca3df49dd8fbf78553d91eacdde74